### PR TITLE
Add v1 preproc scripts

### PR DESCRIPTION
**Pull request by jessb0t**
@apoly1
Here are the three preprocessing scripts that we have been working on.  We will ultimately need to adjust all input/output paths to use relative references before the scripts can be run on the HPC.  This is something that @F-said can help us with!

Also, please note that I modified some of the nomenclature.  As we have been developing these scripts, I found a way to offer a bit more transparency and consistency in the output variables. 🤓 

Please take a look and see if everything appears to be in order from your perspective!  Thank you!

### CODE REVIEW

File: dccs/dccs_preprocessing_v1.R [hunk 1]
```diff
@@ -0,0 +1,410 @@
...
+    testDatTrim_anySwitchOPPCorr <- testDatTrim_anySwitchOPP[testDatTrim_anySwitchOPP$key_resp_13.corr==1, ]
+
+    #for correct trials, for both switch and nonswitch groups, compute mean RT (raw and log-corrected), standard error, and median RT
+    anySwitchCorr_meanRT <- mean(testDatTrim_anySwitchCorr$key_resp_13.rt)
+    anySwitchOPPCorr_meanRT <- mean(testDatTrim_anySwitchOPPCorr$key_resp_13.rt)
+
+    anySwitchCorr_seRT <- sd(testDatTrim_anySwitchCorr$key_resp_13.rt)/sqrt(length(testDatTrim_anySwitchCorr$key_resp_13.rt))
+    anySwitchOPPCorr_seRT <- sd(testDatTrim_anySwitchOPPCorr$key_resp_13.rt)/sqrt(length(testDatTrim_anySwitchOPPCorr$key_resp_13.rt))
+
+    anySwitchCorr_logMeanRT <- mean(log((1+testDatTrim_anySwitchCorr$key_resp_13.rt)))
```
Comment (apoly1) [inline dccs/dccs_preprocessing_v1.R:188]: I was wondering if there is a reason for double "((" after log or is it a typo?
I am sure it is insignificant and make no difference if it a typo, but I was curious about it

File: dccs/dccs_preprocessing_v1.R [hunk 2]
```diff
@@ -0,0 +1,410 @@
...
+
+
+    #TOSHAPE
+    #switch defined as above, independent calculations for switch/nonswitch and switch/preswitch
+
+    #subset the data for switch, preswitch, and nonswitch trials
+    testDatTrim_toShapePairs <- subset(testDatTrim, (testDatTrim$cue=="SHAPE" | testDatTrim$nextCue=="SHAPE"))
+    if(testDatTrim_toShapePairs$cue[1]=="SHAPE"){
+      testDatTrim_toShapePairs <- testDatTrim_toShapePairs[2:length(testDatTrim_toShapePairs$cue),] #remove first row because no COLOR trial follows final SHAPE trial
+    }
```
Comment (apoly1) [inline dccs/dccs_preprocessing_v1.R:207]: sorry I am unsure why no COLOR trial follows final SHAPE trial, could you please explain how this works?

**Review by apoly1 — APPROVED** [review]
The code looks great! DCCS analysis is a huge beautiful beast

File: dccs/dccs_preprocessing_v1.R (hunk 1 above)
Comment (jessb0t) [reply dccs/dccs_preprocessing_v1.R:188]: Looks wholly unnecessary!  Oops!

File: dccs/dccs_preprocessing_v1.R [hunk 3]
```diff
@@ -0,0 +1,410 @@
...
+    #subset the data for switch versus nonswitch trials, create new dataframes for each
+    testDatTrim_anySwitch <- testDatTrim[testDatTrim$switchStatus, ]
+    testDatTrim_anySwitchOPP <- testDatTrim[!testDatTrim$switchStatus, ]
+
+    #compute mean accuracy and standard error for switch versus nonswitch trials
+    anySwitch_meanACC <- mean(testDatTrim_anySwitch$key_resp_13.corr)
+    anySwitchOPP_meanACC <- mean(testDatTrim_anySwitchOPP$key_resp_13.corr)
+
+    anySwitch_seACC <- sd(testDatTrim_anySwitch$key_resp_13.corr)/sqrt(length(testDatTrim_anySwitch$key_resp_13.corr))
+    anySwitchOPP_seACC <- sd(testDatTrim_anySwitchOPP$key_resp_13.corr)/sqrt(length(testDatTrim_anySwitchOPP$key_resp_13.corr))
```
[the remark this answers is not in the document]
Comment (jessb0t) [reply dccs/dccs_preprocessing_v1.R]: No particular reason!

File: dccs/dccs_preprocessing_v1.R (hunk 2 above)
Comment (jessb0t) [reply dccs/dccs_preprocessing_v1.R:207]: If the final trial of the set is SHAPE, then there will be no COLOR trial after it with which to compare.

**Pull request merged**